inception-pdf-editor/src/main/js/pdfanno/src/core/src/render/renderKnob.js

```diff
@@ -48,7 +40,17 @@ function adjustPoint (x, y, radius) {
       if (distance1 < distance2) {
         good = false
```

Comment:
I don't think this will work as expected. The new function `change()` cannot access the variable `good`. This code will fail at runtime.